mozilla-lockbox/lockbox-android
# Minor UI changes for item detail toast and learn more link colors

Fixes #267

Also starts but doesn't finish #262 - I just wanted to get sane colors in place for testing/usability, will wait to see what the final view/result built at #232 ends up so we can easily re-use

## Testing and Review Notes

- open app
- open item detail
- observe the background color of the text link at the bottom matches the screen (grey, not white)
- tap copy button on username/password, observe the white on grey colors of the toast

## Screenshots or Videos

<img width="268" alt="screen shot 2018-12-13 at 8 35 33 am" src="https://user-images.githubusercontent.com/49511/49949781-b55a5f80-feb3-11e8-9b17-8d29c56f8978.png">

<img width="279" alt="screen shot 2018-12-13 at 8 35 23 am" src="https://user-images.githubusercontent.com/49511/49949782-b55a5f80-feb3-11e8-8eb6-df102ca00d68.png">

## To Do

- ~add “WIP” to the PR title if pushing up but not complete nor ready for review~
- ~add unit tests~
- consider running this branch in a debug simulator and check for memory leak notifications or any warnings
